src/Sylius/Behat/Resources/config/services/contexts/ui.xml

```diff
@@ -115,6 +115,11 @@
             <argument type="service" id="sylius.behat.page.admin.payment_method.update" />
             <argument type="service" id="sylius.behat.current_page_resolver" />
             <argument type="service" id="sylius.behat.notification_checker" />
+            <argument type="collection">
```

Comment:
I don't know how :/ There is `sylius.gateway_factories` parameter, but it contains factories types and translation keys, not names. It would be hard to handle it well.